src/OpenAL/OpenTK.OpenAL/Context.cs

```diff
@@ -0,0 +1,9 @@
+namespace OpenTK.OpenAL
+{
+    /// <summary>
+    /// Opaque handle to an OpenAL context.
+    /// </summary>
+    public struct Context
```

Comment:
I don't know if it works like that. But my suggestion would be that it contains atleast an IntPtr, which in memory should then be the same as using a IntPtr directly?

Reply:
That would not work - in this case, it gets translated into a pointer to just something - it's a typed `void*`, essentially. If we added a field, then we'd pass it without specifying it as a pointer. I'm exploring that too right now, however, I'm thinking of another solution. I'm averse to IntPtr right now just because it drops some type safety.

If you have a look in OpenTK.Core, there's a struct called ContextHandle - I've been thinking of adapting that to a more generic solution.